crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -541,6 +541,27 @@ en:
       active: 'Active'
       active_repo_tooltip: 'Managed nodes will be automatically configured to use repositories only if they are marked as active'
 
+  installer:
+    status:
+      installation_failed: 'Installation failed prematurely. Please examine /var/log/crowbar/install.log'
+    index:
+      installation_header: 'Installing Crowbar'
+      installation_successful: 'Installation was successful. You will be redirected in a few seconds'
+      reinstall_notice: 'If you want to install again please remove /opt/dell/crowbar_framework/crowbar-installed-ok'
```

Comment:
I would rephrase to: "If you want to reinstall to get a fresh Crowbar setup, please remove the file ..."

(I know the string was probably already like this in the script, btw)